app/experimenter/nimbus-ui/src/components/FormExperimentOverviewPartial/index.tsx

```diff
@@ -33,31 +46,56 @@ const FormExperimentOverviewPartial = ({
   const nameValidated = (
     name: string,
     // This could be ValidationRules from react-hook-form/validator, but
-    // register() has several signatures and only this one produces a ref 
-    validateRule: Parameters<typeof register>[1] = { required: "Required" },
+    // register() has several signatures and only this one produces a ref
+    validateRule: Parameters<typeof register>[1] = {
+      required: "This field may not be blank.",
+    },
```

Comment:
UX thought: currently if you click outside of the focused field even briefly it turns red and yells at you (it also shifts the page). Would it be possible to move field validation to form submit instead of field blur?

Reply:
Can we push that to a follow-up issue? Will be a point or two of reworking tests for that and this PR's gotten pretty big as-is.

Reply:
Oh of course, it might also be worth asking the design folks, they might disagree with me.